Error: Failed to restore the optimization data package

failed in job: https://ci2.dot.net/job/dotnet_coreclr/job/perf/job/master/job/perf_scenarios_Windows_NT_x64_min_opt_ryujit/1207/

failed log:
C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj : error NU3004: Package 'System.Collections.Immutable 1.3.1' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
  Installing Newtonsoft.Json 10.0.3.
  Installing NETStandard.Library 2.0.0.
C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj : error NU3004: Package 'System.Net.NameResolution 4.3.0' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj : error NU3004: Package 'Microsoft.NETCore.DotNetHostPolicy 2.0.0' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj : error NU3004: Package 'Newtonsoft.Json 10.0.3' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj : error NU3004: Package 'NETStandard.Library 2.0.0' from source 'https://api.nuget.org/v3/index.json': This repository indicated that all its packages are repository signed; however, this package is unsigned.
  Restore failed in 27.81 sec for C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\tool-runtime\project.csproj.

C:\J\w\perf_scenario---af7a1e00>set RESTORE_ERROR_LEVEL=1 
ERROR: An error occured when running: '"C:\J\w\perf_scenario---af7a1e00\.dotnet\dotnet.exe" restore "C:\J\w\perf_scenario---af7a1e00\packages\microsoft.dotnet.buildtools\3.0.0-preview4-03913-01\lib\\tool-runtime\project.csproj" --no-cache --packages "C:\J\w\perf_scenario---af7a1e00\packages" --source https://dotnet.myget.org/F/roslyn/api/v3/index.json --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json  --source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json --source https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json'. Please check above for more details.
BUILD: Error: Failed to restore the optimization data package.
Build step 'Execute Windows batch command' marked build as failure

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.1.503"
+    "dotnet": "2.1.504"
   },
   "native-tools": {
     "cmake": "3.11.1",

